coop_env.py

```diff
@@ -88,7 +89,7 @@ def _get_total_reward(self):
     def _step(self, actions):
         """Execute the specified list of actions.
         """
-        assert self.action_space.contains(actions)
+        # assert self.action_space.contains(actions)
```

Comment:
Don't know why but the assert is failing. I'm inputting
`[[ 0.18617561 -0.08306558 -0.10316727 -0.07944543 -0.08218598 -0.21963407]
 [ 0.63731024  0.41731838  0.46267995  0.28341859  0.28401189  0.43171765]]`

but it doesn't seem to want to eat it.